### DISCUSSION TITLE
Can we return to infrastructure construction?

### DISCUSSION THREAD

**Discussion by medz** [zed-industries/zed#16489] [Feature Requests]
Before today, I have been using Zed for a year. It was great to write TypeScript with it until I needed to maintain my Dart/Flutter and Vue projects.

There are endless problems with language or framework support extensions, and every day I am fighting wits between Zed and LSP.

I recently found that Zed's route is very strange, and I have been polishing Zed itself. I think this is a self-illusion utopia.

Sorry, I recently gave up using Zed because I was tired of encountering LSP problems every day. It seems that the Zed team is very slow to fix or even has no progress in expanding Zed.

To be honest, Zed users need an excellent editor like Zed, but they need an editor with an ecosystem.

I once wanted to make any contribution to Zed, and I also reported bugs before. When I tried to fix the extended LSP for Zed, I found it difficult for Rust beginners like me.

I sincerely hope that Zed will be better, but at least Zed users should be able to use Zed normally.

> I'm back to VS Code, the only thing I have to put up with is that VS Code renders like crap at high resolutions.

**Comment by 0x2CA** [comment]
I also think that the infrastructure is barely operational and has many problems. The main problem is that there are too few people on the team, trying to give editors a unique selling point (accountable to investors?).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
